tests/test_expr_ast.py

```diff
@@ -150,6 +150,18 @@ def test_terminal_get_position():
         assert item in position.items()
 
 
+def test_terminal_get_text():
+    # Given
+    code = "(2 + 2) + 894654"
+
+    # When
+    result = parse(code)
+    text = result.right.get_text()
```

Comment:
Maybe also check `get_text()` for `result` and `result.right`?

Reply:
Agree! Fixed